pkg/virtctl/root.go

```diff
@@ -18,10 +18,18 @@ import (
 	"kubevirt.io/kubevirt/pkg/virtctl/vnc"
 )
 
+var programName string
+
 func NewVirtctlCommand() *cobra.Command {
+	elements := strings.Split(os.Args[0], "/")
```

Comment:
Do you want `filepath.Base(os.Args[0])`?

Reply:
Thanks for the tip!